jupyterhub/apihandlers/users.py

```diff
@@ -344,6 +412,16 @@ def get(self, name, token_id):
         if not user:
             raise web.HTTPError(404, "No such user: %s" % name)
         token = self.find_token_by_id(user, token_id)
+        self.eventlog.record_event(
+            eventlogging_schema_fqn('token-action'),
+            1,
+            {
+                'action': 'get',
+                'target_user': user.name,
```

Comment:
should target_user always have the same schema (`{name, admin}`)?

Reply:
should we do the same for requester everywhere as well?